helm/node-health-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: node-health-psp
+rules:
+  - apiGroups:
+      - extensions
+    resources:
+      - podsecuritypolicies
```

Comment:
i think you can drop the psp stuff, with the psp. You'll probably want to add support for getting nodes

Reply:
Configured nodes.